packages/dialect-react-ui/components/Chat/index.tsx

```diff
@@ -1,69 +1,45 @@
-import { useEffect } from 'react';
+import React from 'react';
```

Comment:
You don't need the full `React` import, `jsx-runtime` gets added in compile time.

Reply:
good catch!